utils/cosmos/iscn/sign.ts

```diff
@@ -47,18 +48,26 @@ export function formatISCNTxPayload(payload: ISCNRegisterPayload): ISCNSignPaylo
         ? `https://like.co/${likerIds[i]}`
         : authorUrls[i][0] || authorName
 
-      const identifiers = authorWallets[i].map((a: any) => ({
+      let identifiers = authorWallets[i].map((a: any) => ({
           '@type': 'PropertyValue',
           propertyID: WALLET_TYPE_REPLACER[a.type] || a.type,
           value: a.address,
         }))
 
+      if ( !identifiers ) {
+        identifiers = likerIds.map((a: any) => ({
+          '@type': 'PropertyValue',
+          propertyID: 'Liker ID',
+          value: `https://like.co/${a}`,
+        }))
+      }
+
       const sameAsArray = authorUrls[i].filter(a => !!a)
       const isNonEmpty = url || authorName || identifiers.length
       if (isNonEmpty) {
         stakeholders.push({
           entity: {
-            '@id': identifiers.length ? identifiers[0].value : url,
+            '@id': identifiers.length ? identifiers[0].value : likerIdsAddress[i] || url,
```

Comment:
Confusing code here, I think we should just store `wallet` as a variable and use `wallet || url` here